new cnics logo

achen2401 opened the pull request:
per Slack [convo](https://cirg.slack.com/archives/C06HPFQE256/p1760725630168849)

use the new CNICS logo in header.

See screenshots:

**CNICS**
<img width="803" height="281" alt="Screenshot 2025-10-24 at 11 57 21 AM" src="https://github.com/user-attachments/assets/7a67c0b6-9abf-4592-a4ea-6794620aa20d" />
<img width="1327" height="61" alt="Screenshot 2025-10-24 at 11 57 34 AM" src="https://github.com/user-attachments/assets/d99cbf0b-74e4-4f93-9a9d-859348edc40e" />

**HIV SUCCESS**
<img width="783" height="283" alt="Screenshot 2025-10-24 at 11 58 22 AM" src="https://github.com/user-attachments/assets/528c8fc5-9203-47e3-9ff2-f8df13361237" />
<img width="875" height="49" alt="Screenshot 2025-10-24 at 11 58 12 AM" src="https://github.com/user-attachments/assets/06592d85-41e1-43a2-b567-297a47001d52" />

remove text "clinical" from databases selection dropdown:
<img width="583" height="176" alt="Screenshot 2025-10-24 at 11 42 27 AM" src="https://github.com/user-attachments/assets/b55cdda6-d58b-4f90-99fb-fec5370e8e8a" />

Commit:
new cnics logo

mcjustin commented:
Thanks @achen2401 . I think we don't that "v3.11.3" tag - I don't know where it came from or what it applies to, but it's not CNICS.

Commit:
remove leaf version text per feedback

achen2401 commented:
> I think we don't that "v3.11.3" tag - I don't know where it came from or what it applies to, but it's not CNICS.

Thank you @mcjustin  for noticing that. I think that is the leaf application version (see [here](https://github.com/uwcirg/leaf/blob/33e25a6b270cc536773921f800fdfe52b3b9050c/src/ui-client/src/containers/Attestation/Attestation.tsx#L154))  I have removed that for now and updated the screenshot.

mcjustin commented:
> > I think we don't that "v3.11.3" tag - I don't know where it came from or what it applies to, but it's not CNICS.
> 
> Thank you @mcjustin for noticing that. I think that is the leaf application version (see [here](https://github.com/uwcirg/leaf/blob/33e25a6b270cc536773921f800fdfe52b3b9050c/src/ui-client/src/containers/Attestation/Attestation.tsx#L154)) I have removed that for now and updated the screenshot.

Ah, thanks @achen2401 - we probably do want that Leaf version number displayed somewhere, just not right next to the CNICS logo. 

As I compare with the HIV Success version (screenshot below), I see that it has the version string in the same place.

The "powered by Leaf [leaf icon]" below the HIV SUCCESS logo is nice. I expect we want that for CNICS. I realize that it's part of that same image for the HIV Success deploy. If it was separate we'd have more flexibility to move position things.

How about this:
`[Project logo image - CNICS | HIV SUCCESS w/ mention of Leaf]`
_powered by Leaf_ `[leaf logo image]` `[version number]`


<img width="814" height="306" alt="image" src="https://github.com/user-attachments/assets/474615af-9b85-451c-a8fa-c19e9f05fac2" />

Commit:
fixes based on feedback

achen2401 commented:
thank you @mcjustin I have put back the Leaf app version along with the text and Leaf image based on your comment.  Please see the updated screenshot.

mcjustin commented:
> thank you @mcjustin I have put back the Leaf app version along with the text and Leaf image based on your comment. Please see the updated screenshot.

Thanks @achen2401 , looks good to me. 
@rpkyle wdyt?

ivan-c commented on src/ui-client/public/images/logos/apps/CNICS/logo_dark.png:
This PR entirely replaces the HIV Success logo with one for CFAR

@rpkyle @mcjustin 
Does this logo work for both projects? Will we ever update the HIV Success deploy of leaf (leaf.cirg.uw.edu) again?

If both projects are active at the same time, we'll probably want this logo to be project-specific

mcjustin replied:
Thanks for flagging @ivan-c  . We'll certainly be updating HIV Success Leaf in the future. 
@achen2401  can you please make this project-specific?

achen2401 replied:
is there a project specific configuration variable, e.g. site id, that the frontend can look for?

mcjustin replied:
> is there a project specific configuration variable, e.g. site id, that the frontend can look for?

I for one don't know... @ivan-c or @rpkyle ?

achen2401 replied:
@ivan-c @mcjustin @rpkyle 
I have changed the code so the logo will be loaded from project specific site ID folder, e.g. HIV_SUCCESS or CNICS.  However, the following environment variable will need to be set: `REACT_APP_SITE_ID`. Otherwise the default Leaf logo will be loaded.

ivan-c replied:
thanks Amy!

just to close the loop, I added that config in this PR

uwcirg/cnics-leaf-deploy/pull/3

Commit:
project specific logos

rpkyle commented:
> > thank you @mcjustin I have put back the Leaf app version along with the text and Leaf image based on your comment. Please see the updated screenshot.
> 
> Thanks @achen2401 , looks good to me. @rpkyle wdyt?

Thanks @achen2401 @mcjustin Looks good to me-- will ask Bridget to weigh in also.

Commit:
changes based on feedback

achen2401 commented:
@mcjustin @rpkyle 
I updated the screenshots to reflect the discussion from the meeting:
- NIDA logo is not present for CNICS
- remove "clinical" text from database dropdown menu
